net/dnsfallback/dnsfallback.go

```diff
@@ -1,10 +1,10 @@
 // Copyright (c) Tailscale Inc & AUTHORS
 // SPDX-License-Identifier: BSD-3-Clause
 
-//go:generate go run update-dns-fallbacks.go
-
 // Package dnsfallback contains a DNS fallback mechanism
 // for starting up Tailscale when the system DNS is broken or otherwise unavailable.
+// The JSON file dns-fallback-servers.json is updated by update-dns-fallbacks.go
+// in this same directory.
```

Comment:
maybe leave `go run ./update-dns-fallbacks.go` so that its easily copy-pasteable